packages/components/text/src/Text.types.ts

```diff
@@ -18,80 +25,79 @@ export type TextWeight = 'regular' | 'medium' | 'semibold' | 'bold';
 /**
  * Text props, based off of the standard react-native TextProps with some new extensions
  */
-export type TextProps<TBase = ITextProps> = TBase &
-  FontVariantTokens & {
-    /**
-     * foreground text color
-     */
-    color?: ColorValue;
+export interface TextProps extends IWithPressableOptions<ITextProps & FontVariantTokens> {
```

Comment:
Could you make sure the SPEC is updated to have the correct values?

Reply:
Done!